Release 0.1.10 breaks support for stable toolchain

Compiling [bevy](https://crates.io/crates/bevy) which uses your crate, I run into an error with stable Rust (1.46.0) since the release of 0.1.10 on crates.io:
```
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /home/akien/.cargo/registry/src/github.com-1ecc6299db9ec823/const-random-macro-0.1.10/src/lib.rs:1:1
  |
1 | #![feature(proc_macro_quote)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Pinning to 0.1.8 fixes it.

*Edit:* I know that using the nightly toolchain would fix it, but I'm reporting this in case you didn't intend to make `const-random-macro` a nightly-only crate.

--- a/macro/src/lib.rs
+++ b/macro/src/lib.rs
@@ -1,6 +1,3 @@
-#![feature(proc_macro_quote)]
-extern crate proc_macro;
-
 use getrandom;
 use proc_macro::*;
 use proc_macro_hack::proc_macro_hack;
@@ -13,6 +10,11 @@ fn gen_random<T>() -> T {
     let mut out = [0u8; 16];
     getrandom::getrandom(&mut out).unwrap();
     unsafe { std::mem::transmute_copy(&out) }
+}
+
+/// Create a TokenStream of an identifier out of a string
+fn ident(ident: &str) -> TokenStream {
+    TokenTree::from(Ident::new(ident, Span::call_site())).into()
 }
 
 #[proc_macro_hack]
@@ -30,11 +32,19 @@ pub fn const_random(input: TokenStream) -> TokenStream {
         "i128" => TokenTree::from(Literal::i128_suffixed(gen_random())).into(),
         "usize" => {
             let value: TokenStream = TokenTree::from(Literal::u128_suffixed(gen_random())).into();
-            quote!(($value as usize))
+            let type_cast: TokenStream = [value, ident("as"), ident("usize")]
+                .iter()
+                .cloned()
+                .collect();
+            TokenTree::from(Group::new(Delimiter::Parenthesis, type_cast)).into()
         }
         "isize" => {
             let value: TokenStream = TokenTree::from(Literal::i128_suffixed(gen_random())).into();
-            quote!(($value as isize))
+            let type_cast: TokenStream = [value, ident("as"), ident("isize")]
+                .iter()
+                .cloned()
+                .collect();
+            TokenTree::from(Group::new(Delimiter::Parenthesis, type_cast)).into()
         }
         _ => panic!("Invalid integer type"),
     }

